src/scripts/underground/Underground.ts

```diff
@@ -337,10 +377,15 @@ $(document).ready(() => {
 namespace Underground {
     export const BASE_ENERGY_MAX = 50;
     export const BASE_ITEMS_MAX = 3;
+    export const BASE_ITEMS_MIN = 1;
     export const BASE_ENERGY_GAIN = 3;
     export const BASE_ENERGY_REGEN_TIME = 60;
     export const BASE_DAILY_DEALS_MAX = 3;
     export const BASE_BOMB_EFFICIENCY = 10;
+    export const BASE_SURVEY_CHARGE_EFFICIENCY = 2; //1
```

Comment:
Can we not just have this as 1?
```suggestion
    export const BASE_SURVEY_CHARGE_EFFICIENCY = 1;
```

Reply:
I can't recall 100% why, but there's a reason for it.

Either it displays incorrectly in the upgrade screen or it calculates incorrectly.

Reply:
Fixed this.